Test-Rack-Software/util/git_commands.py

```diff
@@ -18,11 +18,11 @@ def run_script(arg_list):
     @arg_list {List[str]} -- A list of arguments to execute the command with
     @description remote_command.py is a wrapper for GitPy, which leaks memory"""
     print(
-        "(git_commands) Running script [python remote-command.py " +
+        f"(git_commands) Running script [{cfg.python_root}python remote-command.py " +

```

Comment:
Change this to cfg.python instead of python root, and just point to python in general